### ISSUE TITLE
My Dead Corpse arrow remains after resurrect.

### ISSUE THREAD

**Issue by SalviaSage** [AeroScripts/QuestieDev#384]
My Dead Corpse arrow remains after resurrect.

Fix?

**Comment by Dyaxler** [comment]
Working on it.

**Comment by Dyaxler** [comment]
Ah yes... in this edge case... if you have the quest arrow disabled and died, it wouldn't hide the arrow after resurrect even if you have the corpse arrow enabled because (QuestieConfig.corpseArrow == false). Bad logic...

```
            if (QuestieConfig.arrowEnabled == false) and (QuestieConfig.corpseArrow == false) then
                TomTomCrazyArrow:Hide()
            end
```

Re-engineered the PLAYER_UNGHOST event and OnUpdate code logic... fix will be in the 3.70 release.

**Comment by Muehe** [comment]
New beta release is out. It can be downloaded [here](https://github.com/AeroScripts/QuestieDev/releases/tag/v3.7.0-beta).
